Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FINUFFT"
 uuid = "d8beea63-0952-562e-9c6a-8e8ef7364055"
 author = "Ludvig af Klinteberg <ludvigak@kth.se>"
-version = "1.1.0"
+version = "1.3.0"
```

Comment:
I suggest bump to 2.0.0, since we are changing the interface

Reply:
Sounds reasonable, done!